Automattic/wp-calypso
# Media: Replace getNextPageQuery flux selector with redux

#### Changes proposed in this Pull Request

* Add new redux selectors for next page query
* Track the next page handle in the `fetching` selector (is there a better place for this?)
* Use the new redux selectors for the next page query rather than flux
* Remove the flux stuff

#### Testing instructions



* Using a site with very many media items (more than 50) ensure infinite scroll still works and that the initial page load is 20 items
* Do the same for pexels and google photos
* Also because I made changes to the `getMediaItem` selector, ensure that adding media items to posts in the classic editor still works and that the images appear when you reload the page after saving the draft or publishing (this should be sufficient to test those changes).

A note about Google Photos: the service is sometimes not working correctly (which you can see in the response from the API) so it's worth comparing with prod.

Part of https://github.com/Automattic/wp-calypso/issues/43663
